test/Frontend/debug-diagnostic-names.swift

```diff
@@ -0,0 +1,23 @@
+// RUN: not %target-swift-frontend -debug-diagnostic-names -typecheck %s 2>&1 | %FileCheck %s --check-prefix=CHECK_NAMES
+// RUN: not %target-swift-frontend -typecheck %s 2>&1 | %FileCheck %s --check-prefix=CHECK_NONAMES
+
+let x =
+// CHECK_NAMES: error: expected initial value after '=' [expected_init_value]
+// CHECK_NONAMES: error: expected initial value after '='
```

Comment:
Just to be sure it's not printing the value, can you add a `{{$}}` at the end to match the end of the line?

Reply:
Good call, added